resonite/egg-pterodactyl-resonite.json

```diff
@@ -1,30 +1,30 @@
 {
-    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "update_url": null,
-        "version": "PTDL_v2"
+        "version": "PTDL_v2",
+        "update_url": null
     },
-    "exported_at": "2024-06-01T00:04:38+00:00",
+    "exported_at": "2025-01-07T18:34:26+11:00",
     "name": "Resonite",
     "author": "espeon@espeon.dev",
     "description": "Enter a novel digital universe with infinite possibilities. Whether you resonate with people around the world in a casual conversation, playing games and socializing or you riff off each other when creating anything from art to programming complex games, you'll find your place here.",
     "features": null,
     "docker_images": {
-        "Mono": "ghcr.io/parkervcp/yolks:mono_latest"
+        ".NET 9": "ghcr.io\/pelican-eggs\/yolks:dotnet_9"
```

Comment:
This Image is not made for Steam games. i would use steamcmd:dotnet.

but this Image is still on dotnet 8

Reply:
Sent in a PR to add .NET 9 support to steamcmd:dotnet and will update these eggs accordingly.

Reply:
Eggs updated in https://github.com/pelican-eggs/games-steamcmd/pull/117/commits/979944a7dfda464ff4a1794c40d8c190f00859a8

Reply:
Dotnet 9 was added to steamcmd:dotnet